src/elements/common/_variables.scss

```diff
@@ -45,7 +45,8 @@ $sidebarTabResponsiveSize: 48px;
 }
 
 @mixin placeholder {
+    color: $bdl-gray-62;
     font-weight: normal;
     font-family: $font;
-    opacity: .6;
+    opacity: 1; // https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder#opaque_text
```

Comment:
tldr: some browsers set an opacity less than 100% which affects the text color

Reply:
Should we change the opacity for certain browsers then? Would that make the placeholder in the browsers that have 100% opacity different than before?

Reply:
the main culprit seems to be Firefox. its default opacity for placeholders is `0.54`: https://hg.mozilla.org/mozilla-central/file/a07aebef20e7/layout/style/forms.css#l180

by setting `opacity: 1;`, all browsers will use the same opacity for placeholder text (i.e. align Firefox with Chrome and Edge)